src/MakieGallery.jl

```diff
@@ -78,6 +78,12 @@ function load_database(files::AbstractVector{<: AbstractString})
     database
 end
 
+const REFIMGDIR
+
+function __init__()
+    global REFIMGDIR
```

Comment:
You should use a `const` without an additional global declaration...And then just use a Ref to be able to update the global!

Reply:
oops, I forgot to remove that looks like.  I didn't mean for it to be a CONST - since you could theoretically use it in scripts as well.

Reply:
Thanks for pointing that out - I had it as non-const, but messed around a little too much with my git history.